fix_demographic_jspf.sh

```
@@ -0,0 +1,13 @@
+cat << 'INNER_EOF' > src/main/webapp/WEB-INF/jspf/demographic-field-length-limits.jspf.patch
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)